### PR TITLE
chore: tune ci and release

### PR DESCRIPTION
**Pull request by PsiACE**
I hereby agree to the terms of the CLA available at: https://databend.rs/dev/policies/cla/

## Summary

- functions: codegen-units = 16 , 9m -> 7m on my local
- disable grov via comments

Before merging, let's observe the unit tests.

## Changelog

- Not for changelog (changelog entry is not required)

## Related Issues

Fixes #issue

## Test Plan

Unit Tests

Stateless Tests

### CODE REVIEW

**Comment by PsiACE** [comment]
```
 ---- servers::http::http_query_handlers::test_http_service_tls_server_mutual_tls_failed stdout ----
Error: Code: 1045, displayText = Address already in use (os error 98).
```

A flaky test? let's re-run it again.

**Comment by Xuanwo** [comment]
> ```
> Error: Code: 1045, displayText = Address already in use (os error 98).
> ```

As I recognized it more than three times, I think we should investigate it. Maybe we should listen to random ports during unit tests to avoid conflict?

File: .github/actions/test_unit/action.yml [hunk 1]
```diff
@@ -37,24 +37,24 @@ runs:
...
-        RUSTDOCFLAGS: '-Zprofile -Ccodegen-units=1 -Cinline-threshold=0 -Clink-dead-code -Coverflow-checks=off -Zpanic_abort_tests'
+        # RUSTFLAGS: '-Zprofile -Ccodegen-units=1 -Cinline-threshold=0 -Clink-dead-code -Coverflow-checks=off -Zpanic_abort_tests --cfg tokio_unstable'
+        # RUSTDOCFLAGS: '-Zprofile -Ccodegen-units=1 -Cinline-threshold=0 -Clink-dead-code -Coverflow-checks=off -Zpanic_abort_tests'
 
-    - name: Install grcov
-      uses: actions-rs/cargo@v1
-      with:
-        command: install
-        args: grcov
+    # - name: Install grcov
```
Comment (Xuanwo) [inline .github/actions/test_unit/action.yml:43]: Please leave a comment here to decide whether uncomment or remove them totally in the future.

Comment (PsiACE) [reply .github/actions/test_unit/action.yml:50]: Okay, I've added the instructions for uncomment and remove.

**Review by sundy-li — APPROVED** [review]

**Comment by BohuTANG** [comment]
The failure test issued already:
https://github.com/datafuselabs/databend/issues/4118

**Pull request merged**